src/futures/Cargo.toml

```
@@ -1,14 +1,14 @@
 [package]
-name = "engula-runtime"
+name = "engula-futures"
 version = "0.3.0"
 edition = "2021"
 license = "Apache-2.0"
 homepage = "https://engula.io"
 repository = "https://github.com/engula/engula"
-description = "The Engula asynchronous runtime."
+description = "Asynchronous utilities for Engula."
 
 [dependencies]
 futures = "0.3.19"
 
 [dev-dependencies]
-tokio = { version = "1.15.0", features = ["full"] }
+tokio = "1.15.0"
```

```suggestion
tokio = { version = "1.15.0", features = ["full"] }
```

We need `"macros", "rt"` features for running tests individually. Since it's a dev dependency, we can simply set features to `"full"`.